src/detectors/FEMC/FEMC.cc

```diff
@@ -46,12 +46,21 @@ void InitPlugin(JApplication* app) {
   decltype(CalorimeterHitDigiConfig::resolutionTDC) EcalEndcapP_resolutionTDC =
       10 * dd4hep::picosecond;
   const double EcalEndcapP_sampFrac = 0.029043; // updated with ratio to ScFi model
-  decltype(CalorimeterHitDigiConfig::corrMeanScale) EcalEndcapP_corrMeanScale =
-      fmt::format("{}", 1.0 / EcalEndcapP_sampFrac);
+  decltype(CalorimeterHitDigiConfig::corrMeanScale) EcalEndcapP_corrMeanScale = "1.0";
```

Comment:
I think, the old version was correct – it simulated the fact that gain is adjusted between the configurations to have a similar dynamic range.

Reply:
My original point was that the lightYield stuff happens before corrMeanScale is applied in digi.

Reply:
I don't think that's the case. The formulas are supposed to divide the sampled number of photons by the expected number of photons again to convert energy back to GeV scale, so that will multiply by sampFrac once to cancel one of them.